editor/plugins/texture_editor_plugin.cpp

```diff
@@ -44,6 +44,15 @@ TextureRect *TexturePreview::get_texture_display() {
 	return texture_display;
 }
 
+void TexturePreview::set_texture_display_outline_width(int p_width) {
+	outline_width = p_width;
```

Comment:
The only call to `set_texture_display_outline_width` looks like this
```
set_texture_display_outline_width(outline_width);
```
so you are calling this method with a property, and then it sets this property again. Looks like the method can be removed?

Reply:
@KoBeWi I added this as public method for future possible uses, but for now it could be removed for sure.

Reply:
Yeah we don't add things "for future use", unless it's for something specific you plan to do in the future. The method can be easily added when needed.